tutorials/plot_ica_from_raw.py

```diff
@@ -31,7 +31,7 @@
            filter_length='10s', phase='zero-double')
 
 ###############################################################################
-# 1) Fit ICA model using the FastICA algorithm
+# 1) Fit ICA model using the FastICA algorithm.
```

Comment:
Do you think it's worth adding some segment as bad with Annotations so people can see it, or overkill in this example?